styles/header.css

```diff
@@ -0,0 +1,79 @@
+.header {
+  display: flex;
+  height: 48px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.left-section {
+  margin-left: 12px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+}
+
+.logo {
+  color: #6070ff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  padding: 10px 12px;
+  text-decoration: none;
+}
+
+.right-section {
+  margin-right: 24px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+  border: none;
+  background-color: white;
+}
+
+.icon-menu {
+  padding: 8px;
+  width: 24px;
+  height: 24px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  margin-left: 24px;
+  margin-right: 24px;
+  background-image: url('/Module-1/My-portfolio/images/header-shapes\ mobile.png');
+  background-repeat: no-repeat;
```

Comment:
- [ ] This image is not been accessed and as a result, it is not displaying, I tried to fix the space between the last slash in the dev tools but the problem still persists. Kindly take a second look at it.

### What I have here as your design

![image](https://user-images.githubusercontent.com/86200356/205259430-f3f47f6d-43ad-41a3-8c02-43b05fa01010.png)

Reply:
Fixed.

Reply:
Thank you very much